Northstar.Client/mod/scripts/vscripts/ui/menu_ingame.nut

```diff
@@ -0,0 +1,702 @@
+//global function InitLobbyStartMenu
```

Comment:
New modified vanilla script. Needs to be first manually committed to `main` under Respawn user by a mainainer

(DO NOT RESOLVE THIS UNTIL THE FILES HAVE BEEN ADDED TO MAIN)

Reply:
Done in 80ceb2b317336c454d83392028501a31da89e77e